wire-ios-data-model/Source/ManagedObjectContext/CoreDataStack+Backup.swift

```diff
@@ -188,6 +189,36 @@ extension CoreDataStack {
         )
     }
 
+    fileprivate static func ingest(
+        mpBackupFile: URL,
+        onFailure fail: (CoreDataStack.BackupImportError) -> ()
+    ) {
+        // TODO: Figure out the actual self-user domain before importing
+        let importer = MPBackupImporter(selfUserDomain: "wire.com")
+        let result = importer.import(multiplatformBackupFilePath: mpBackupFile.path())
```

Comment:
I would have expected the `import` to be asynchronous. Is it blocking?

Reply:
It is blocking at the moment.

I am considering making it async as well in the next iterations.

Reply:
It would look something like this:
https://github.com/rickclephas/KMP-NativeCoroutines?tab=readme-ov-file#swift-concurrency